api/API_VERSIONING.md

```diff
@@ -28,9 +29,24 @@ configuration resources (e.g. bootstrap, xDS resources such as `Cluster`). The
 v3 API bootstrap configuration is `envoy.config.bootstrap.v3.Boostrap`, even
 though it might might transitively reference `envoy.service.trace.v2`.
 
+The minor version enables API stability and support, and to safely deprecate old
+features and reduce the tech debt. A client can support a range of minor API
+versions, and can request xDS resources that match this range, using the resource
+[context parameters](https://github.com/cncf/udpa/blob/master/xds/core/v3/context_params.proto).
+Deprecated features, from previous minor or major versions (and their code), can be safely removed
+from the client. In general, within a minor API version, we do not allow any breaking changes
+(more details below). The minor version will be reset to 0 upon a release of a new major version.
```

Comment:
"any breaking changes to the Protobuf definitions for the API".

Reply:
Done.